extra_foam/services.py

```diff
@@ -137,6 +139,20 @@ def start_redis_server():
             logger.error(f"Failed to config the Redis server.\n" + repr(e))
             sys.exit(0)
 
+        # Increase the hard and soft limits for the redis client pubsub buffer
+        # to 512MB and 128MB, respectively.
+        cli_buffer_cfg = (client.config_get("client-output-buffer-limit")[
+            "client-output-buffer-limit"]).split()
+        assert len(cli_buffer_cfg) == 12
+        soft_limit = 128 * 1024 ** 2  # 128 MB
+        soft_second = 60  # in second
```

Comment:
what is `soft_second` limit here?

Reply:
If the buffer size is above the `soft_limit` for `soft_second`, the client will be disconnected. For `hard_limit`, the client will be immediately disconnected.